src/main/java/seedu/address/logic/commands/FindRoomCommand.java

```diff
@@ -12,7 +12,7 @@
 public class FindRoomCommand extends Command {
     public static final String COMMAND_WORD = "findroom";
     public static final String MESSAGE_NO_EMPTY_ROOM = "There are unfortunately no more empty rooms!";
-    private static final String NUMBER_OF_ROOMS_UNDEFINED = "Please define number of rooms";
+    public static final String NUMBER_OF_ROOMS_UNDEFINED = "Please define number of rooms";
```

Comment:
Just to clarify, this exception is thrown if the user did not call the command addnumberofrooms right?

Reply:
Yes